qutebrowser/browser/urlmarks.py

```diff
@@ -141,12 +141,21 @@ class QuickmarkManager(UrlMarkManager):
         - removed gets emitted with the name as argument.
     """
 
+    def _quickmark_directory(self):
+        inconfigdir = os.path.join(standarddir.config(), 'quickmarks')
+        indatadir = os.path.join(standarddir.data(), 'quickmarks')
+        if os.path.isfile(inconfigdir):
+            return standarddir.config()
+        else:
+            return standarddir.data()
+
+
```

Comment:
flake8 (code style checker) is going to complain about two instead of one line here (between methods) :wink: